File: mario.c
#include <stdio.h>
#include <cs50.h>

void printLine(int row, int lenght){
    for (int i=0; i<lenght-row; i++){
        printf(" ");
    }
    
    for (int i=0; i<row; i++){
        printf("#");
    }
    
    printf("  ");
    
    for (int i=0; i<row; i++){
        printf("#");
    }
    printf("\n");
}

int main(void){
    int n;
    
    do{
        n = get_int("Height: ");
    } while (n<1 || n>8);
    
    for(int i=1; i<= n; i++)
        printLine(i, n);
    
}

File: mario.py
from cs50 import get_int

def printline(row, size):
    for i in range(size - row):
        print(" ", end="")
    for i in range(row):
        print("#", end="")
    print("  ", end="")
    for i in range(row):
        print("#", end="")
    print()

while True:
    size = get_int("Enter pyramid size: ")
    if size>0 and size<9:
        break
for row in range(1, size+1):
    printline(row, size)
        
